main.nf

```diff
@@ -328,6 +328,7 @@ process get_software_versions {
     picard MarkDuplicates --version &> v_markduplicates.txt  || true
     samtools --version &> v_samtools.txt
     multiqc --version &> v_multiqc.txt
+    edgeR --version &> v_edgeR.txt
```

Comment:
@silviamorins This wont work because `edgeR` isnt available on the command-line as an explicit command i.e. you run it from within R. I havent done this for R packages before but a similar command for python is:
https://github.com/nf-core/atacseq/blob/c65bf6cd11c8a98807084d2d4039822a19f03969/main.nf#L1644

Maybe worth testing it locally first. 

PS: You will also have to update:
https://github.com/nf-core/rnaseq/blob/dev/bin/scrape_software_versions.py

Reply:
I see, I suspected something like that.. I'm looking around, thank you both!